az-core/src/main/java/azkaban/Constants.java

```diff
@@ -33,12 +33,16 @@
 public class Constants {
 
   // Azkaban Flow Versions
-  public static final String AZKABAN_FLOW_VERSION_2_0 = "2.0";
+  public static final String AZKABAN_FLOW_VERSION = "Azkaban-Flow-Version";
+  public static final Double VERSION_2_0 = 2.0;
```

Comment:
why not just double instead of Double class?

Reply:
Will change to double, thanks!